studiopress/genesis-custom-blocks
# Don't display the editor form if there's no editor field

#### Changes
* If there are only Inspector fields, this doesn't display the editor form:
![the-ins](https://user-images.githubusercontent.com/4063887/108272677-323e8f00-7138-11eb-8e06-da7d863f83af.gif)
* Branched off https://github.com/studiopress/genesis-custom-blocks/pull/63. That accounts for most of this PR's diff. Here's the [diff](https://github.com/studiopress/genesis-custom-blocks/compare/6a2f8f692c3d34d61c7ac0271dd05213985c6a04...834cbd4c3e4df4cf4cbc40cf8b35fc8e986f0a29) for this feature only.

#### Testing instructions
1. `/wp-admin` > Custom Blocks > Create New
2. Add a field with the location of 'Inspector'
3. Publish the block
4. Create a new post
5. Add the block
6. Expected: the `<ServerSideRender>` displays, as shows in the `.gif` above. The block form never displays.
